src/js/routes/factories/network.js

```diff
@@ -111,7 +111,8 @@ let RouteFactory = {
                 parentCrumb: 'virtual-networks-tab-detail-tasks-detail',
                 getCrumbs: function () { return []; }
               }
-            }
+            },
+            title: 'Details'
```

Comment:
The title is actually 'content'. That doesn't belong into a definition imo. But just my 2€¢.